core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/database/base/DatabaseType.java

```diff
@@ -14,5 +14,7 @@
     REGISTRY,
     HIVE_DAS,
     HUE,
-    STREAMS_MESSAGING_MANAGER
+    STREAMS_MESSAGING_MANAGER,
+    PROFILERMETRIC,
```

Comment:
Would fit the naming convention better.
PROFILER_METRIC
PROFILER_AGENT

Reply:
Make sense. I will try to incorporate this change.